libnymea-core/integrations/thingmanagerimplementation.h

```diff
@@ -189,6 +194,10 @@ private slots:
     QHash<ThingClassId, ThingClass> m_supportedThings;
     QHash<ThingId, Thing*> m_configuredThings;
     QHash<ThingDescriptorId, ThingDescriptor> m_discoveredThings;
+    QHash<QString, Logger*> m_stateLoggers;
+    QHash<QString, Logger*> m_actionLoggers;
+    QHash<QString, Logger*> m_eventLoggers;
+    Logger *m_logger;
```

Comment:
= nullptr;

Reply:
acutally not used (yet). I've dropped it completely. The idea was to log thing creation/removal/setupState etc... But I'll leave that for the future.